docs/developers-guide/versioning.md

```diff
@@ -31,33 +29,32 @@ v0.46.3.1
 
 ### Major
 
-We release major version when we introduce new features or breaking changes.
+We release major versions when we introduce new features or breaking changes. Check out our [release announcements](https://www.metabase.com/releases).
 
 ### Point
 
 Sometimes called a minor release, we issue point releases when we add bug fixes and refinements to existing features.
 
 ### Hotfix
 
-Sometimes called a patch release, we issue these hotfix releases to fix security issues in a timely manner, or to undo a horrific regression.
+Sometimes called a patch release, we issue these hotfix releases to fix security issues as soon as we can.
```

Comment:
This is misleading: we don't issue patch releases to fix security issues. We automatically publish them every day